### PR TITLE
Update blocklistConfig.json with CPBL hosts

### PR DESCRIPTION
**Pull request by shuvashish76**
**Host link:** https://raw.githubusercontent.com/bongochong/CombinedPrivacyBlockLists/master/newhosts-final.hosts
**Source:** [CombinedPrivacyBlockLists](https://github.com/bongochong/CombinedPrivacyBlockLists) by @bongochong
**License:** [CPAL-1.0](https://github.com/bongochong/CombinedPrivacyBlockLists/blob/master/LICENSE)
**Description:** Ad & malware-blocking hosts files, IP blocklists, PAC filters, and ABP / uBO subscriptions, all merged, sorted and de-duped from multiple reputable sources, along with bongochong's own research. It block malicious and harmfully deceptive content, like advertising, tracking, telemetry, scam, and malware servers. The lists **do not** block porn, social media, or so-called "fake news" domains, except for the advertising, tracking, telemetry, scam, and malware servers associated with such sites. False positives are rigorously removed from all lists upon discovery, and the hosts files are processed by an IDN-to-Punycode conversion routine (meaning that they can be utilized on all operating systems)

### CODE REVIEW

**Comment by ignoramous** [comment]
@bongochong can you comment if you're okay with RethinkDNS' use of these lists? Thanks.

**Comment by bongochong** [comment]
Amazing timing (was just about to head to work, then got this notification) but absolutely, yes. Also @shuvashish76 the lists are now WTFPL, and thanks for finding some of my content useful :) Looking forward to trying it out with ReThink!

**Comment by shuvashish76** [comment]
@ignoramous @bongochong thanks for the quick reply :D 

I just clicked on license here: https://github.com/bongochong/CombinedPrivacyBlockLists/blob/master/LICENSE
I see WTFPL: http://www.wtfpl.net/txt/copying/

**Comment by bongochong** [comment]
Yep! Dual license repo for now (CPAL for scripts, WTFPL for lists), as @ignoramous brought up some very good points in regard to that. Many thanks guys.

File: blocklistConfig.json [hunk 1]
```diff
@@ -1224,6 +1224,13 @@
       "group": "privacy",
       "subg": "services",
       "url": "https://raw.githubusercontent.com/ftpmorph/ftprivacy/master/blocklists/hola-luminati-full-block.txt"
+     },
+     {
+      "vname": "CPBL Final (bongochong)",
+      "format": "hosts",
+      "group": "security",
```
Comment (ignoramous) [inline blocklistConfig.json:1231]: `Security`? Or `Privacy`?

File: blocklistConfig.json [hunk 2]
```diff
@@ -1224,6 +1224,13 @@
...
+      "subg": "piracy",
```
Comment (ignoramous) [inline blocklistConfig.json:1232]: Are you sure this `subg` is correct?

File: blocklistConfig.json (hunk 1 above)
Comment (shuvashish76) [reply blocklistConfig.json:1231]: CPBL blocks "Ad & malware-blocking hosts files, IP block lists, PAC filters" etc...
So you are right it match to our description of `Privacy` : Block adware, spyware, scareware, and trackers.

Screenshot_2021-08-11-21-13-00

**Pull request merged**